modules/op-restarting-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,64 @@
+// This module is included in the following assemblies:
+// * pac/managing-pipeline-runs-pac.adoc
+
+:_content-type: PROCEDURE
+[id="restarting-pipeline-run-using-pipelines-as-code_{context}"]
+= Restarting a pipeline run using {pac}
+
+You can restart a pipeline run with no events, such as sending a new commit to your branch or raising a pull request. To restart all or specific pipeline runs, use the *Re-run all checks* feature in a GitHub App or use the following commands:
+
+* The `/test` and `/retest` comment restarts all pipeline runs.
+
+* The `/test <pipelinerun-name>` and `/retest <pipelinerun-name>` comment restarts a specific pipeline run.
+
+* The `/cancel` comment cancels all pipeline runs.
+
+* The `/cancel <pipelinerun-name>` comment cancels a specific pipeline run.
+
+The results of the commands are visible under the *Checks* tab of a GitHub App.
+
+.Procedure
+
+* If you use a GitHub App, go to the *Checks* tab and click *Re-run all checks*.
+
+* If you target a pull or merge request, use the comments inside your pull request:
++
+.Example
+[source]
+----
+This is a comment inside a pull request.
+/retest
+----
+
+* If you target a push request, include the comments within your commit messages.
++
+[NOTE]
+====
+This feature is supported for the GitHub provider only.
+====
+
+.. Go to your GitHub repository.
+
+.. Click the *commits* section.
+
+.. Click the commit where you want to restart a pipeline run.
+
+.. Click on the line number where you want to add a comment.
++
+.Example
+[source]
+----
+This is a comment inside a commit.
+/retest <pipelinerun-name>
+----
++
+[NOTE]
```

Comment:
Yes, I wanted it to be nested there. It made more sense to me to put it to the step where user is making the comment rather than the whole procedure. Thank you!